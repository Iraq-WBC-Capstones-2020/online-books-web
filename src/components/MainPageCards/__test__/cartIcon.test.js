import React from 'react';
import renderer from 'react-test-renderer';
import CartIcon from './../cartIcon';

test('cartIcon snapshot', () => {
  const tree = renderer.create(<CartIcon />).toJSON();
  expect(tree).toMatchSnapshot();
});
