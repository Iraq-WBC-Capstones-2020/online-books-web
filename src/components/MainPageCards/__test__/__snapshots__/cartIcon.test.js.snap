// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cartIcon snapshot 1`] = `
<svg
  className="inline-block"
  fill="none"
  height="21"
  viewBox="0 0 20 21"
  width="20"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M9.16671 8.19041H10.8334V5.69041H13.3334V4.02374H10.8334V1.52374H9.16671V4.02374H6.66671V5.69041H9.16671V8.19041ZM5.83337 15.6904C5.39135 15.6904 4.96742 15.866 4.65486 16.1786C4.3423 16.4911 4.16671 16.915 4.16671 17.3571C4.16671 17.7991 4.3423 18.223 4.65486 18.5356C4.96742 18.8481 5.39135 19.0237 5.83337 19.0237C6.2754 19.0237 6.69932 18.8481 7.01188 18.5356C7.32445 18.223 7.50004 17.7991 7.50004 17.3571C7.50004 16.915 7.32445 16.4911 7.01188 16.1786C6.69932 15.866 6.2754 15.6904 5.83337 15.6904ZM14.1667 15.6904C13.7247 15.6904 13.3008 15.866 12.9882 16.1786C12.6756 16.4911 12.5 16.915 12.5 17.3571C12.5 17.7991 12.6756 18.223 12.9882 18.5356C13.3008 18.8481 13.7247 19.0237 14.1667 19.0237C14.6087 19.0237 15.0327 18.8481 15.3452 18.5356C15.6578 18.223 15.8334 17.7991 15.8334 17.3571C15.8334 16.915 15.6578 16.4911 15.3452 16.1786C15.0327 15.866 14.6087 15.6904 14.1667 15.6904ZM5.97504 12.9821L6.00004 12.8821L6.75004 11.5237H12.9584C13.5834 11.5237 14.1334 11.1821 14.4167 10.6654L17.6334 4.82374L16.1834 4.02374H16.175L15.2584 5.69041L12.9584 9.85708H7.10837L7.00004 9.63208L5.13337 5.69041L4.34171 4.02374L3.55837 2.35708H0.833374V4.02374H2.50004L5.50004 10.3487L4.37504 12.3904C4.24171 12.6237 4.16671 12.8987 4.16671 13.1904C4.16671 13.6324 4.3423 14.0564 4.65486 14.3689C4.96742 14.6815 5.39135 14.8571 5.83337 14.8571H15.8334V13.1904H6.18337C6.07504 13.1904 5.97504 13.0987 5.97504 12.9821Z"
    fill="#BBE1FA"
  />
</svg>
`;
